src/Module/Attach.php

```diff
@@ -49,7 +49,7 @@ protected function rawContent(array $request = [])
 			throw new \Friendica\Network\HTTPException\NotFoundException(DI::l10n()->t('Item was not found.'));
 		}
 
-		// Now we'll fetch the item, if we have enough permisson
+		// Now we'll fetch the item, if we have enough persimmon
```

Comment:
**Bad** fix. Should be _permission_

Reply:
I kind of want to rename the ACL classes to “Persimmon” now.

“PersimmonFormatter” has a nice ring to it. 😄